src/proxyServer/index.js

```diff
@@ -33,10 +33,10 @@ function addExternalMappingsInterceptor() {
 }
 
 function initProxyServer() {
```

Comment:
hoxy takes the CA here, but in the other local server it takes the server CERT, this is because hoxy rewrites the CERTS. Where as the node server we use to server webpack, just needs the server CERT.